scripts/verify-version-alignment.sh

```
@@ -0,0 +1,101 @@
+  local hostport
+  hostport="$(docker port "$cid" "${container_port}/tcp" | head -n1 | sed 's/.*://')"
+  if [[ -z "$hostport" ]]; then
+    echo "  ✗ could not resolve published host port for ${container_port}/tcp" >&2
+    failures=$((failures + 1))
+    return
+  fi
+
+  local url="http://127.0.0.1:${hostport}${path}"
+  local body="" actual=""
+  local attempt
+  for attempt in $(seq 1 30); do
```

Shellcheck (0.11.0)

[warning] attempt appears unused. Verify use (or export if used externally).

(SC2034)